expat/README.md

```diff
@@ -37,6 +37,7 @@ GNU autoconf and libtool tools.  To do this, you need to have
 autoconf 2.58 or newer. Run the script like this:
 
 ```console
+cd expat
```

Comment:
Good to have but not related to this pull request. Please make that a dedicated pull request.